app/src/main/java/io/apicurio/registry/rules/RulesServiceImpl.java

```diff
@@ -61,17 +66,26 @@ public void applyRules(String artifactId, ArtifactType artifactType, ContentHand
             rules = storage.getGlobalRules();
             useGlobalRules = true;
         }
-        if (rules.isEmpty()) {
-            return;
+

```

Comment:
I think this should remain - if there are no rules to apply we want to skip the `UPDATE` section below.

Reply:
Good spot, have put it back!